### PR TITLE
Replace 2.4/2.5 with 2.6 for `use_reentrant=None`

### PR DESCRIPTION
**Pull request by nzw0301**
In my understanding, still, pytorch hasn't raised an error when specifying `use_reentrant=None` for gradient checkpointing yet. So this PR replaces `2.5` with `2.6` that is the version I assume, so might be wrong. Similarly fixes `warning` block's torch version about this future behaviour.

### CODE REVIEW

**Review by soulitzer — APPROVED** [review]
Thanks!

**Comment by soulitzer** [comment]
@pytochbot merge

**Comment by nzw0301** [comment]
Thank you for your review. The command's account looks a typo. Could you try merging again?

**Review by soulitzer — APPROVED** [review]
